src/components/curator-actions/BountyCardDetails.svelte

```diff
@@ -157,3 +163,7 @@
 		{/if}
 	</div>
 </div>
+
+{#if curatorsDialogOpen}
+	<Signatories bind:open={curatorsDialogOpen} {bounty} curatorAddress={bounty.curator} />
+{/if}
```

Comment:
```suggestion
<Signatories bind:open={curatorsDialogOpen} {bounty} curatorAddress={bounty.curator} />
```
I believe it’s generally fine to just have them included